edge-util/test/Microsoft.Azure.Devices.Edge.Util.Test.Common/TestUtilities.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+namespace Microsoft.Azure.Devices.Edge.Util.Test.Common
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Text;
+    using Xunit;
+
+    public static class TestUtilities
+    {
+        public static void ApproxEqual(double expected, double actual, double tolerance)
```

Comment:
make it as extension of Assert class.

Reply:
I tried that first, you can't make an extension function on a static class